src/charm.py

```diff
@@ -349,6 +387,28 @@ def _relation_created(self, relation_name: str) -> bool:
         """
         return bool(self.model.relations[relation_name])
 
+    @property
+    def _gnb_name(self) -> str:
+        """The gNB's name contains the model name and the app name.
+
+        Returns:
+            str: the gNB's name.
+        """
+        return f"{self.model.name}-gnbsim-{self.app.name}"
+
+    def _get_tac_as_int(self) -> Optional[int]:
+        """Convert the TAC value in the config to an integer.
+
+        Returns:
+            TAC as an integer. None if the config value is invalid.
+        """
+        tac = None
+        try:
+            tac = int(self.model.config.get("tac"), 16)  # type: ignore[arg-type]
+        except ValueError:
+            logger.info("Invalid TAC value in config: it cannot be converted to integer.")
```

Comment:
Suggesting to log in error level as this is an invalid config which needs to get attention